server/postgreSchema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE SCHEMA ReservationsModule;
+
+CREATE TABLE ReservationsModule.Restaurants (
+  id int not null unique,
+  name str,
```

Comment:
Don't think PostgreSQL doesn't support 'str', you should look into using char, varchar, or text